linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_usergroup_modification/oval/shared.xml

```diff
@@ -97,7 +97,7 @@
   </ind:textfilecontent54_test>
   <ind:textfilecontent54_object id="object_audit_rules_usergroup_modification_etc_shadow_auditctl" version="1">
     <ind:filepath>/etc/audit/audit.rules</ind:filepath>
-    <ind:pattern operation="pattern match">^\-w[\s]+/etc/shadow[\s]+\-p[\s]+\b([rx]*w[rx]*a[rx]*|[rx]*a[rx]*w[rx]*)\b[\s]+(-k[\s]+|-F[\s]+key=)\w+[\s]*$</ind:pattern>
```

Comment:
I believe the main problem with the `-k` option here was that the regex `\w+` doesn't accept the hyphen character `-`, and in some cases it might happen that hyphens are used, then the regex would fail. The non-whitespace regex would be preferable in these cases IMO `\S+`

Reply:
@ggbecker I'm not sure what mean here, where are you heading?